tests/phpunit/WC_Stripe_Intent_Controller_Test.php

```
@@ -548,6 +550,156 @@ public function test_mandate_options_for_card_setup_intent_for_subscription() {
+
+		echo wp_json_encode( $response );
+
+		// Clean up
+		$order->delete();
+		WC_Stripe_Order_Helper::set_instance( $original_helper );
+		Ajax_Test_Helper::remove_hooks();
+
+		$this->assertNotNull( $response, 'Controller output should be valid JSON. Output: ' . $output );
+		$this->assertTrue( $response['success'] );
+		$this->assertArrayHasKey( 'return_url', $response['data'] );
+	}
```

PHPMD (2.15.0)

[error] test_update_order_status_ajax_returns_redirect_url_after_acquiring_lock accesses the super-global variable $_POST. (undefined)

(Superglobals)